### ISSUE TITLE
error if chart have one label

### ISSUE THREAD

**Issue by reimax** [chartist-js/chartist#570]
demo from site:

new Chartist.Line('.ct-chart', {
  labels: ['Monday'],
  series: [
    [12],

  ]
}, {
  fullWidth: true,
  chartPadding: {
    right: 40
  }
});

the only point is to the left of the chart

**Comment by reimax** [comment]
in setting i have option fullWidth: true. if delete this, chart work correct

**Comment by nhagen** [comment]
I'm also having this issue.

```
new Chartist.Line('.ct-chart', {
  labels: ['Monday'],
  series: [[1]]
}, { fullWidth: true });
```

spits out http://i.imgur.com/hZdXooQ.png

**Comment by joaolavoier** [comment]
I also have the same issue, this is the line code of issue: https://github.com/gionkunz/chartist-js/blob/develop/src/scripts/axes/step-axis.js#L27

A solution to this issue: 

```
var calc = (options.ticks.length - (options.stretch ? 1 : 0));
this.stepLength = this.axisLength / (calc === 0 ? 1 : calc);
```

**Comment by gionkunz** [comment]
this should be fixed as of 0.9.6
